src/main/java/uk/ac/ebi/ega/accession/user/rest/AccessioningUserRestController.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ega.accession.user.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.ega.accession.user.AccessioningUser;
+import uk.ac.ebi.ega.accession.user.AccessioningUserRepository;
+
+@RestController
+@RequestMapping(value = "/users")
+public class AccessioningUserRestController {
+
+    @Autowired
+    private AccessioningUserRepository accessioningUserRepository;
+
+    @RequestMapping(path = "addOrUpdate", method = {RequestMethod.POST},
```

Comment:
RESTful APIs are based on the HTTP method only, so we shouldn't extend the URL with this `addOrUpdate`.